llvm/lib/Transforms/Vectorize/VPlanRecipes.cpp

```diff
@@ -291,6 +291,49 @@ InstructionCost VPRecipeBase::computeCost(ElementCount VF,
   llvm_unreachable("subclasses should implement computeCost");
 }
 
+InstructionCost VPSingleDefRecipe::computeCost(ElementCount VF,
```

Comment:
Did this accidentally get added during the update to current main? Currently the branch doesn't build, because there are now multiple definitions of VPSingleDefRecipe::computeCost: https://buildkite.com/llvm-project/github-pull-requests/builds/117409#019307ae-fff8-4fcd-9f4a-94b4296ef3bd